### DISCUSSION TITLE
How to merge anything if fast test is always failing

### DISCUSSION THREAD

**Discussion by pheepa** [ClickHouse/ClickHouse#62383] [Q&A] [answered]
Here is my pr https://github.com/ClickHouse/ClickHouse/pull/62262

There have been a lot of checks and fast test is always failing (random tests have connection refused error). When I ran failed test locally, the pass successfully. Furthermore, these should not be affected by my pr because I committed just one new array function.
I know it can be fixed in upstream commits in repo and I have to sync fork before I commit, but even so, there is no luck.

**Comment by Algunenano** [comment, marked as answer]
> How to merge anything if fast test is always failing

I would recommend you to stop rebasing and merging master and instead look into why the tests are crashing the server with your changes. As there are multiple tests running in parallel, if the server crashes then random tests will fail because of it (the ones being run when it crashed).

Check the logs in https://s3.amazonaws.com/clickhouse-test-reports/62262/2211e011277e0fa9ad7d4923330de144b96455c2/fast_test.html

It's crashing. If you look at the server logs and look for `<Fatal>` you'll find something like:
```
2024.04.09 16:42:22.974419 [ 185286 ] {} <Fatal> BaseDaemon: (version 24.4.1.1, build id: 2E89AC3E924B35A75F8046D03EFDF6A8FAF3F5AE, git hash: 1097498787048cd1111cbd244b120c7384ed7ac5) (from thread 179023) (query_id: 35390e62-e716-47c0-a9a9-127d387d8ef5) (query: SELECT [toUInt16(1)] AS arr1, [toUInt32(1)] AS arr2, round(arrayJaccardIndex(arr1, arr2), 2);) Received signal Segmentation fault (11)
```

The PR can't be merged until that's fixed.

**Reply by pheepa** [reply]
Sorry for million rebases... I will stop

Didn't know that fail of one can be the reason of the fail of another
